lib/iaas/AzureClient.js

```diff
@@ -154,7 +154,10 @@ class AzureClient extends BaseCloudClient {
           .snapshots
           .deleteMethodAsync(this.settings.resource_group, snapshotName)
           .then(retval => logger.info(`Deleted snapshot ${snapshotName}`, retval))
-          .catch(err => logger.error(`Error occured while deleting snapshot ${snapshotName}`, err));
+          .catch(err => {
```

Comment:
Just for my knowledge , wouldnt it be better if this catch be for the outer [try](https://github.com/muralisc/service-fabrik-broker/blob/5b07c0bd66fbf122dfc36f37cb5578f7b34f1f7b/lib/iaas/AzureClient.js#L152) ?

Reply:
to my understanding in the current code above its the same .
also  we need a catch in the innermost promise chain ?